webapi/src/main/scala/org/knora/webapi/messages/admin/responder/valueObjects/ProjectsValueObjectsADM.scala

```diff
@@ -0,0 +1,189 @@
+      Validation.fail(BadRequestException(LONGNAME_MISSING_ERROR))
+    } else {
+      Validation.succeed(new Longname(value) {})
+    }
+
+  def make(value: Option[String]): Validation[Throwable, Option[Longname]] =
+    value match {
+      case None    => Validation.succeed(None)
+      case Some(v) => self.make(v).map(Some(_))
+    }
+}
+
+/**
+ * ProjectDescription value object.
+ */
+sealed abstract case class ProjectDescription private (value: Seq[StringLiteralV2])
+object ProjectDescription { self =>
+  def make(value: Seq[StringLiteralV2]): Validation[Throwable, ProjectDescription] =
+    if (value.isEmpty) {
+      Validation.fail(BadRequestException(PROJECT_DESCRIPTION_MISSING_ERROR))
+    } else {
+      Validation.succeed(new ProjectDescription(value) {})
+    }
+
+  def make(value: Option[Seq[StringLiteralV2]]): Validation[Throwable, Option[ProjectDescription]] =
+    value match {
+      case Some(v) => self.make(v).map(Some(_))
+      case None    => Validation.succeed(None)
+    }
+}
+
+/**
+ * Project Keywords value object.
+ */
+sealed abstract case class Keywords private (value: Seq[String])
+object Keywords { self =>
+  def make(value: Seq[String]): Validation[Throwable, Keywords] =
+    if (value.isEmpty) {
+      Validation.fail(BadRequestException(KEYWORDS_MISSING_ERROR))
+    } else {
+      Validation.succeed(new Keywords(value) {})
+    }
+
+  def make(value: Option[Seq[String]]): Validation[Throwable, Option[Keywords]] =
+    value match {
+      case Some(v) => self.make(v).map(Some(_))
+      case None    => Validation.succeed(None)
+    }
+}
+
+/**
+ * Project Logo value object.
+ */
+sealed abstract case class Logo private (value: String)
+object Logo { self =>
+  def make(value: String): Validation[Throwable, Logo] =
+    if (value.isEmpty) {
+      Validation.fail(BadRequestException(LOGO_MISSING_ERROR))
+    } else {
+      Validation.succeed(new Logo(value) {})
+    }
+  def make(value: Option[String]): Validation[Throwable, Option[Logo]] =
+    value match {
+      case None    => Validation.succeed(None)
+      case Some(v) => self.make(v).map(Some(_))
+    }
+}
+
+/**
+ * ProjectSelfjoin value object.
+ */
+sealed abstract case class ProjectSelfJoin private (value: Boolean)
+object ProjectSelfJoin { self =>
+  def make(value: Boolean): Validation[Throwable, ProjectSelfJoin] =
+    Validation.succeed(new ProjectSelfJoin(value) {})
+
+  def make(value: Option[Boolean]): Validation[Throwable, Option[ProjectSelfJoin]] =
+    value match {
+      case Some(v) => self.make(v).map(Some(_))
+      case None    => Validation.succeed(None)
+    }
+}
+
+/**
+ * ProjectStatus value object.
+ */
+sealed abstract case class ProjectStatus private (value: Boolean)
+object ProjectStatus { self =>
+  def make(value: Boolean): Validation[Throwable, ProjectStatus] =
+    Validation.succeed(new ProjectStatus(value) {})
+
+  def make(value: Option[Boolean]): Validation[Throwable, Option[ProjectStatus]] =
```

Comment:
As discussed yesterday, this is handy for both optional and non-optional fields. Instead of different name of the other method I just added missing types annotations to make it more visible which method is used for certain field.